### ISSUE TITLE
Question about the model

### ISSUE THREAD

**Issue by pearl-rabbit** [ckczzj/PDAE#4]
Hello,I have read your article and believe it is a great job!
And I have a question，have you ever tried training without using a pre-trained model, that is, adding a gradient estimator directly to the original model for retraining?

**Comment by ckczzj** [comment]
Thanks for your attention!

Learning representation based on a pre-trained model is our fundamental idea, so we didn't try it. If you don't have a pre-trained diffusion model, you can train one using our code, or directly train a conditional diffusion model like Diff-AE. Maybe you actually want a gradient estimator to do something?

Your question is very interesting, and I think it can work. I will try it on toy dataset MNIST and tell you the results these days.

**Comment by pearl-rabbit** [comment]
Okay, thank you for your reply.Just an idea, I don't know if the gradient estimator will have the same effect. Looking forward to your results.

**Comment by ckczzj** [comment]
![image](https://github.com/ckczzj/PDAE/assets/23724208/5853fc31-ab9a-4b7b-9fdc-8aa78da7f11b)

Hello, it works for MNIST.

**Comment by pearl-rabbit** [comment]
Ok.

**Comment by pearl-rabbit** [comment]
Hello, I'm sorry to bother you again. Have you encountered any output nan during the sampling phase? The pretrained model can sample normally, and what is the reason for this? (To clarify, I am not entirely using the code you provided on Github)

**Comment by ckczzj** [comment]
Do you mean the output of gradient estimator may become nan? I haven't encountered it.

**Comment by pearl-rabbit** [comment]
Okay, it could be a problem with my network designed.
